### DISCUSSION TITLE
How We Want to Apply User Stories to our Dev Process

### DISCUSSION THREAD

**Discussion by bengineerdavis** [codebuddies/backend#183] [General]
At the request of @lpatmo @tgrrr and others, I'm starting this here because of our conclusions in [Discussion 171](https://github.com/codebuddies/backend/discussions/171) where we determined that Behavior Driven Development didn't suite a test suite that will likely only focus on unit testing (thanks again for everyone who support this experiment!). But one of the best things that exploration offered was to have a larger discussion about when and where integration tests and BDD **would** best serve the team. Hopefully we can condense as many constructive ideas and critiques here as we can.

**Comment by lpatmo** [comment]
For the /hangouts feature at least, the plan is:

1) Finalize the user stories on https://github.com/codebuddies/frontend/discussions/138, with designs (designs don't need to be high-fidelity)
2) Finalize the needed API requests/responses from that
3) Use the tracking issue Bethany created at https://github.com/codebuddies/backend/issues/160 as a guide to build out the API

There may be some work to write the user stories in a cleaner way, but ideally both BE and FE would reference those user stories in that https://github.com/codebuddies/frontend/discussions/138 issue, imo.
